trainTestReinforcementAlgorithm.py

```diff
@@ -0,0 +1,168 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.13.0
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %%
+import time
+import gym
+import gym_snake
+from stable_baselines3 import A2C
+from stable_baselines3.common.logger import configure
```

Comment:
nit: is this being used? My syntax highlighter says nah and commenting it out everything runs fine.